src/qml/Legend.qml

```diff
@@ -181,12 +182,14 @@ ListView {
       }
 
       QfToolButton {
-        visible: InTracking ? true : false
+        id: trackingBadge
+        property bool isVisible: InTracking ? true : false
+        visible: isVisible
```

Comment:
```suggestion
        visible: InTracking
```
Just like that is not enought?

Reply:
@domi4484 , I'm using that check in multiple places (e.g. to determine the length of the layer name label), and therefore it's much better to have one bool property referred in those multiple places so when I upgrade the property it updates everywhere.